flask_resty/exceptions.py

```diff
@@ -6,6 +6,16 @@
 
 
 class ApiError(Exception):
+    """An API exception. When raised, Flask-RESTy will send an HTTP response with
+    the provided `status_code`. The JSON encoded body contains a list of the
+    provided `errors` nested under a key named ``errors``.
```

Comment:
maybe link to jsonapi for reference?

Reply:
Will do, I actually didn't know about jsonapi before writing these docs - thought i was looking at XPath at first xD

Reply:
flask-resty takes a lot of inspiration from jsonapi. maybe it's something important to say in the initial description